config/locales/hyrax.de.yml

```
@@ -348,6 +348,9 @@ de:
     base:
       citations:
         header: 'Zitate:'
+      form:
+        batch_upload: batch upload
+        create_separate_work_for_each_file: Um eine separate Arbeit für jede Datei zu erstellen, gehen Sie zu
```

For German I would like to make it even more specific by adding: 
```suggestion
        create_separate_work_for_each_file: Um eine separate Arbeit für jede einzelne Datei zu erstellen, gehen Sie zu
```

---

Our german translation was automatically generated

---

This is the common workflow in hyrax https://github.com/samvera/hyrax/wiki/Translations-(internationalization).
Doing a rather good job, Google and Co. fail miserably for very specific context or short phrases/ single words. At least the German part. How is id performing with French?

---

I would appreciate if you include my update in your PR instead of me adding a follow-up PR correcting the German.

---

French translations are very often unusable or even comic. For exemple, "date" is translated by "rendez-vous amoureux" (it means, an appointment with a girl or a boy for whom you have feelings...). It goes beyond a translation problem, the expressions used are not necessarily adapted (for example, we can't us "work" at all). We sometimes need to rewrite the English version! And that prevents us from integrating our French translation propositions into Hyrax.